src/Injector.ts

```diff
@@ -10,11 +10,31 @@ export interface GetChildren {
 	<C>(inject: C, children: DNode[]): DNode[];
 }
 
+/**
+ * The binding mappers for properties and children
+ */
+export interface Mappers {
+	getProperties: GetProperties;
+	getChildren: GetChildren;
+}
+
+/**
+ * Default noop Mappers for the container.
+ */
+export const defaultMappers: Mappers = {
+	getProperties(inject: any, properties: any): any {
+		return {};
```

Comment:
`Object.create(null)` is better in situations where the object is truly only being used as a property map and there is no need to access the larger `Object` functionality, like `.hasOwnProperty()` or related.  It has an appreciable performance improvement in cases where the object has a lot of property lookups into it, as there is no chain to ascend when looking for a key.

As far as a standard, no, I don't think we have ever codified though every once in a while, when I see an empty object literal I wonder if `Object.create(null)` could/should be used.  If we were to codify it, it would be as above, when it is purely being used a strict map/hash.